spec/integration/basic_subscriber_spec.rb

```diff
@@ -21,10 +21,8 @@ def cancelled
 
   context "ActionSubscriber.auto_pop!" do
     it "routes messages to the right place" do
-      channel = connection.create_channel
-      exchange = channel.topic("events")
-      exchange.publish("Ohai Booked", :routing_key => "greg.basic_push.booked")
-      exchange.publish("Ohai Cancelled", :routing_key => "basic.cancelled")
+      ::ActionSubscriber::Publisher.publish("greg.basic_push.booked", "Ohai Booked", "events")
+      ::ActionSubscriber::Publisher.publish("basic.cancelled", "Ohai Cancelled", "events")
```

Comment:
We could switch all the other integration tests to publish this way, but I wanted to keep this PR really small for now and just have a single example of using it in an integration test.